src/borg/archive.py

```diff
@@ -60,6 +60,9 @@ def __init__(self, output_json=False, iec=False):
         self.osize = self.usize = self.nfiles = 0
         self.osize_parts = self.usize_parts = self.nfiles_parts = 0
         self.last_progress = 0  # timestamp when last progress was shown
+        self.files_stats = {"A": 0, "U": 0, "M": 0, "E": 0}
```

Comment:
Hmm, maybe using a `defaultdict(int)` would be better here:

https://docs.python.org/3.9/library/collections.html#collections.defaultdict